### ISSUE TITLE
App doesn't start on OnePlus 7

### ISSUE THREAD

**Issue by michelgrootjans** [covid-be-app/cwa-app-android#92]
This phone is about 1 year old

I've tried:
- uninstall then reinstall the app
- restarting the phone
- uninstall, restart then reinstall

See screenshots
![app-cannot-start-small](https://user-images.githubusercontent.com/345770/95000079-06be6c00-05be-11eb-8f4a-19c033e878d6.jpg)
![phone-specs-small](https://user-images.githubusercontent.com/345770/95000084-0cb44d00-05be-11eb-9e23-d08c1783fe64.jpg)

**Comment by WatcherWhale** [comment]
Have you updated your Google services or have a pending android update?

**Comment by leroynicolasalexi** [comment]
It means google exposure api can be accessed. Did you install the app from the play store ?

**Comment by michelgrootjans** [comment]
Yes. Pure install from the play store.

**Comment by michelgrootjans** [comment]
I managed to solve the problem. The Google Play Store had corrupt data. Clearing the Play Store's cache didn't help. Clearing its app data did, and as far as I can see without any negative side effects.

I found the solution in [this article](https://medium.com/@johnkpdsmith/how-to-fix-error-checking-for-updates-on-google-play-store-d8d4299a086c).

Sorry for the inconvenience. This issue can be closed.
